po/es.po

```diff
@@ -288,53 +280,47 @@ msgstr "`impl Trait`"
 
 #: src/SUMMARY.md src/generics/exercise.md
 msgid "Exercise: Generic `min`"
-msgstr ""
+msgstr "Ejercicio: Genérico `min`"
 
 #: src/SUMMARY.md
 msgid "Day 2: Afternoon"
 msgstr "Día 2: tarde"
 
 #: src/SUMMARY.md src/std-types.md
-#, fuzzy
 msgid "Standard Library Types"
-msgstr "Biblioteca estándar"
+msgstr "Tipos de biblioteca estándar"
```

Comment:
"Tipos de biblioteca estándar" reads like "kinds of standard libraries", where the original meaning is: types that exist inside the standard library. Not sure how to fix.

```suggestion
msgstr "Tipos de la biblioteca estándar"
```